src/Models/BookingStatus.php

```diff
@@ -0,0 +1,570 @@
+<?php
+namespace SturentsLib\Api\Models;
+
+/**
+ * ** This file was generated automatically, you might want to avoid editing it **
+ *
+ * Basic contract information and booking status.
+ */
+class BookingStatus extends SwaggerModel
+{
+	/**
+	 * Full name of the user who initiated the booking
+	 * @var string
+	 */
+	protected $tenant_name = '';
+
+	/**
+	 * Identifier provided in the redirect URL when the user initiated the booking
+	 * @var string
+	 */
+	protected $tenant_id = '';
+
+	/**
+	 * Generated unique identifier, this will be the same as the tenant_id unless the same identifier has been provided for multiple bookings.
+	 * @var string
+	 */
+	protected $unique_tenant_id = '';
+
+	/**
+	 * Email address of the user who initiated the booking
+	 * @var string
+	 */
+	protected $tenant_email = '';
+
+	/**
+	 * Phone number of the user who initiated the booking
+	 * @var string
+	 */
+	protected $tenant_phone = '';
+
+	/**
+	 * 1|0, indicates whether the contract requires a guarantor
+	 * @var float
+	 */
+	protected $guarantor_required = 0.0;
```

Comment:
Given this is a boolean flag, assume this should be an int representing 0 or 1 as the comment suggests?

Reply:
Okay, I see the issue, if this gets generated from the swagger file which only lets us define a 'number' there's no getting around this. Same for all of the other instances.

Reply:
Yeah I don't see how we can change this. I can manually update the fields to be integers but when the next update to the docs is run it will overwrite any changes. 
I had a similar problem in that in the api.yaml I wanted to do: 
```
type: [string, null]
default: null
```
but the GenerateModels.php doesn't accept arrays for the types. 

I wonder whether I could create a new GenerateModels file which extends the vendor version and use that to customise the generate for the BookingStatus model